pywr/_core.pyx

```diff
@@ -1210,6 +1210,7 @@ cdef class VirtualStorage(Storage):
     def __cinit__(self, ):
         self._allow_isolated = True
         self.virtual = True
+        self.active = True
```

Comment:
I think we should set this to `True` in a reset method. Then it ensures it resets to the init value.

Reply:
done